packages/btfs2/karalabe-usb-pthread_barrier.patch.diff

```diff
@@ -0,0 +1,24 @@
+From e188e2d7eefa1a057a20921236f22a6008b29c93 Mon Sep 17 00:00:00 2001
+From: Fredrik Fornwall <fredrik@fornwall.net>
+Date: Thu, 12 Dec 2019 01:07:20 +0100
+Subject: [PATCH] Fix build when targetting Android 7 or later
+
+Support for pthread_barrier was introduced in Android 7, so trying to
+build a custom implementation of that clashes with the system headers.
```

Comment:
Hmm, this patch has been merged in the karalabe repo, but I guess the dependency has not yet been updated in go-btfs. Not sure why I removed it..